### PR TITLE
TM-894: add translations to plugin

### PR DESCRIPTION
**Pull request by michalsikora**

### CODE REVIEW

File: app/config/parameters-ci.yml.dist [hunk 1]
```diff
@@ -36,3 +36,5 @@ parameters:
       service_options: []
 
     sentry_dsn:
+    plugins_dir:
+      - '%kernel.root_dir%/../src/SevenTag/Plugin'
```
Comment (kallosz) [inline app/config/parameters-ci.yml.dist]: we don't need this right now. You can hard code this currently. We move this to config file when we will be creating plugin bundle.

File: app/config/parameters.yml.dist [hunk 2]
```diff
@@ -34,5 +34,9 @@ parameters:
       enabled: false
       service: ~
       service_options: []
-
+      
     sentry_dsn:
+    
+    plugins_dir:
```
Comment (kallosz) [inline app/config/parameters.yml.dist]: same as above

File: src/SevenTag/Api/AppBundle/Controller/TranslationsController.php [hunk 3]
```diff
@@ -46,14 +46,19 @@ public function indexAction()
...
 
-        if (!is_object($locale)) {
+        if (is_null($locales)) {
             throw new \LogicException(
                 sprintf('The file for %s language was not found', $lang)
             );
         }
 
-        return json_decode(file_get_contents($locale), true);
+        foreach ($locales as $locale) {
```
Comment (kallosz) [inline src/SevenTag/Api/AppBundle/Controller/TranslationsController.php]: move this to separate service. Controller shouldn't be aware of how to merge translation mechanism.

File: web/app_dev.php [hunk 4]
```diff
@@ -27,7 +27,7 @@
 // Feel free to delete this, extend it, or make something more sophisticated.
 if (isset($_SERVER['HTTP_CLIENT_IP'])
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
+    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1', '100.0.0.1')) || php_sapi_name() === 'cli-server')
```
Comment (kallosz) [inline web/app_dev.php]: this shouldn't be committed.

Comment (michalsikora) [inline web/app_dev.php]: This is for vagrant/docker users

**Pull request merged**